backend/app/main.py

```
@@ -71,15 +72,14 @@ async def validation_exception_handler(request: Request, exc: RequestValidationE
     configure_middleware(app)
 
     # 2. Register Routers
-    # We import wiring routers from app.router.* which in turn import module routers
+    # We import wiring routers from app.routers.* which in turn import module routers
     app.include_router(auth_wiring.router)
-    
-    # Include PDF processing router (no prefix - proxy handles /api/proxy/...)
-    from modules.pdf_processing.router import router as pdf_router
-    app.include_router(pdf_router, tags=["PDF Processing"])
+    app.include_router(pdf_processing_wiring.router)
```

Ruff (0.14.8)

Undefined name `pdf_processing_wiring`

(F821)